geode-core/src/main/java/org/apache/geode/internal/cache/PersistentOplogSet.java

```diff
@@ -82,8 +86,9 @@
   /** counter used for round-robin logic * */
   private int dirCounter = -1;
 
-  public PersistentOplogSet(DiskStoreImpl parent) {
+  public PersistentOplogSet(DiskStoreImpl parent, PrintStream out) {
     this.parent = parent;
+    this.out = out;
```

Comment:
Should we remove calling system.out and fix the place where it is relaying on system.out.  Or change this to log.info?

Reply:
Yes, I think someone who's working on GFSH should work on the next steps of removing the use of System.out and adding output to the appropriate commands.

This PR just restores the System.out calls and adds a test for it.